src/platform/user/selectors.js

```diff
@@ -1,5 +1,7 @@
 // TODO: perhaps make these selectors fail gracefully if state.user, or any of
 // the properties on the user object are not defined
+import { selectFacilities } from '../../applications/vaos/utils/selectors';
```

Comment:
@jbalboni do you think that we should be using this selector or the platform one?: `selectPatientFacilities` https://github.com/department-of-veterans-affairs/vets-website/blob/9d09c1f5e3515735fa12256ef46c3c0467ff0310/src/platform/user/selectors.js#L12

Why does this selector filter out `742` facilities? https://github.com/department-of-veterans-affairs/vets-website/blob/b453d3de21ac862e8fc0952d0d1488e02d27b214/src/applications/vaos/utils/selectors.js#L42

I'd lean towards the former because it feels wrong for something in `platform` to depend on an app-level selector.

Reply:
Yes, I agree that apps shouldn't depend on code in other app folders, they should depend on platform code.

The reason for the 742 logic is that there's a bug in the range of ids we allow through into that facility array (the facilities array is derived from MVI correlation ids, which aren't all facility related). I was seeing `742V1` come through in the array, which is not a facility, and it was breaking VAOS. This is a short term fix while we figure out the correct range. I have a PR open in the devops repo to update this, but Patrick V also reached out to someone at MHV to give us a definitive answer about what ids we should allow into that facilities array.

I don't think this issue will impact the work on this PR, though, so I think you could safely use the platform selector. I would either do that, or we can move the logic I wrote into the platform selector.

Reply:
ah in that case I'd add that filter to the platform selector with a comment explaining why it's there and a TODO to hopefully remove it at some point.

Reply:
@erikphansen I already refactored to use the platform selector - I don't think it matters for my use case because I am only looking for isCerner and none of the other properties

Reply:
@bradl3yC yes but now the VAOS team's `selectIsCernerOnlyPatient` selector  is using the `platform` selector that _doesn't_ filter out the `742` facilities and will break, right?

Reply:
@erikphansen That's true, this will break our code the way it's written now.